tests/recipes/test_train_step.py

```diff
@@ -479,7 +479,8 @@ def test_train_step_with_tuning_best_parameters(
     assert "penalty" in best_params_yaml
     assert "eta0" in best_params_yaml
 
-    run_id = open(train_step_output_dir / "run_id").read()
+    with open(train_step_output_dir / "run_id") as f:
+        run_id = f.read()
```

Comment:
```suggestion
    run_id = train_step_output_dir.joinpath("run_id").read_text()
```

looks like `train_step_output_dir` is a `Path` object. We can use `read_text`.